### ISSUE TITLE
AntiCapcha service does not support notTrue method

### ISSUE THREAD

**Issue by tixsys** [jumper423/decaptcha#11]
AntiCapcha service does not support notTrue method, but there is such possibility on webpage.

**Comment by jumper423** [comment]
@tixsys 
Thank you for your comment. AntiCaptcha has a method described here [https://anticaptcha.atlassian.net/wiki/spaces/API/pages/48693258/reportIncorrectImageCaptcha+send+complaint+on+an+image+captcha](https://anticaptcha.atlassian.net/wiki/spaces/API/pages/48693258/reportIncorrectImageCaptcha+send+complaint+on+an+image+captcha)in the near future I will implement this functionality.

**Comment by tixsys** [comment]
Yeah, that was i meant that its available as web-api (in that exact link), but not as library method ) any ETA?

Thanks ;)
